src/bentoml/_internal/spark.py

```diff
@@ -0,0 +1,193 @@
+from __future__ import annotations
+
+import typing as t
+import logging
+import os.path
+import tempfile
+from typing import TYPE_CHECKING
+
+import bentoml
+from bentoml import Bento
+from bentoml.exceptions import NotFound
+from bentoml.exceptions import BentoMLException
+from bentoml.exceptions import MissingDependencyException
+from bentoml._internal.tag import Tag
+from bentoml._internal.service.loader import load_bento
```

Comment:
Let's also do relative import here. General rules are any imports under `_internal` should do relative import.